app/packages/plugins/src/index.ts

```diff
@@ -151,13 +156,14 @@ export function usePlugins() {
       })
       .then(() => {
         setState("ready");
+        setPluginsInitialized(true);
```

Comment:
Seems redundant to `setState('ready')`

Reply:
Good point. I should've just externalized the local state instead. Will fix this